incubator-tools/line_item_comparision/line_item_comparision.ipynb

```diff
@@ -0,0 +1,563 @@
+{
```

Comment:
<p>The last section could be reformatted into a bullet point list for readability</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/682/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>